make/test.mk

```diff
@@ -0,0 +1,213 @@
+else
+push-operator-app-registry: get-operator-version push-operator-image
+endif
+	$(eval OPERATOR_MANIFESTS := tmp/manifests/$(TAG))
+	$(Q)operator-courier flatten manifests/devconsole/ $(OPERATOR_MANIFESTS)
+	$(Q)cp -vf deploy/crds/* $(OPERATOR_MANIFESTS)
+	$(Q)sed -i -e 's,REPLACE_IMAGE,$(TEKTONCD_PIPELINE_OPERATOR_IMAGE):$(TAG),' $(OPERATOR_MANIFESTS)/tektoncd-pipeline-operator.v$(TEKTONCD_PIPELINE_OPERATOR_VERSION).clusterserviceversion-v$(TEKTONCD_PIPELINE_OPERATOR_VERSION).yaml
+	$(Q)operator-courier verify $(OPERATOR_MANIFESTS)
+	$(eval QUAY_API_TOKEN := $(shell curl -sH "Content-Type: application/json" -XPOST https://quay.io/cnr/api/v1/users/login -d '{"user":{"username":"'${QUAY_USERNAME}'","password":"'${QUAY_PASSWORD}'"}}' | jq -r '.token'))
+	$(Q)operator-courier push $(OPERATOR_MANIFESTS) $(TEKTONCD_PIPELINE_APPR_NAMESPACE) $(TEKTONCD_PIPELINE_APPR_REPOSITORY) $(TEKTONCD_PIPELINE_OPERATOR_VERSION)-$(TAG) "$(QUAY_API_TOKEN)"
+
+.PHONY: test-operator-source
+test-operator-source: push-operator-app-registry
+	$(eval OPSRC_NAME := tektoncd-pipeline-operators-$(TAG))
+	$(eval OPSRC_DIR := test/operatorsource)
+	$(Q)oc project openshift-marketplace 
+	$(Q)sed -e "s,REPLACE_NAMESPACE,$(TEKTONCD_PIPELINE_APPR_NAMESPACE)," ./$(OPSRC_DIR)/operatorsource.yaml | sed -e "s,REPLACE_OPERATOR_SOURCE_NAME,$(OPSRC_NAME)," | oc apply -f -
+	$(Q)sed -e "s,REPLACE_APPR_REPOSITORY,$(TEKTONCD_PIPELINE_APPR_REPOSITORY)," ./$(OPSRC_DIR)/catalogsourceconfig.yaml | oc apply -f -
+	$(Q)sed -e "s,REPLACE_APPR_REPOSITORY,$(TEKTONCD_PIPELINE_APPR_REPOSITORY)," ./$(OPSRC_DIR)/subscription.yaml | oc apply -f -
+	$(Q)./hack/check-crds.sh
+	$(Q)OPSRC_NAME=$(OPSRC_NAME) \
+	TEKTONCD_PIPELINE_OPERATOR_VERSION=$(TEKTONCD_PIPELINE_OPERATOR_VERSION) \
+	go test -vet off ${V_FLAG} $(shell go list ./... | grep $(OPSRC_DIR)) -failfast
+
+.PHONY: olm-integration-cleanup
+olm-integration-cleanup: get-test-namespace
+ifeq ($(OPENSHIFT_VERSION),3)
+	$(Q)oc login -u system:admin
+	$(Q)-oc delete subscription my-devconsole -n operators
+	$(Q)-oc delete catalogsource my-catalog -n olm
+endif
+ifeq ($(OPENSHIFT_VERSION),4)
+	$(Q)-oc delete subscription my-devconsole -n openshift-operators
+	$(Q)-oc delete catalogsource my-catalog -n openshift-operator-lifecycle-manager
+endif
+	$(Q)-oc delete project $(TEST_NAMESPACE)  --wait
+
+.PHONY: test-e2e-olm-ci
+test-e2e-olm-ci: ./vendor
+	$(Q)sed -e "s,REPLACE_IMAGE,registry.svc.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:tektoncd-pipeline-operator-registry," ./test/e2e/catalog_source_OS4.yaml | oc apply -f -
+	$(Q)oc apply -f ./test/e2e/subscription_OS4.yaml
+	$(eval DEPLOYED_NAMESPACE := openshift-operators)
+	$(Q)./hack/check-crds.sh
+	$(Q)operator-sdk test local ./test/e2e --no-setup --go-test-flags "-v -timeout=15m"
+
+.PHONY: test-e2e-ci
+test-e2e-ci: get-test-namespace ./vendor
+	$(Q)oc new-project $(TEST_NAMESPACE)
+	$(Q)-oc apply -f ./deploy/crds/devconsole_v1alpha1_component_crd.yaml
```

Comment:
we don't these crds.
we can either change it to our CRD or remove the targets which are not being tested at present